client/src/app/pages/applications/application-assessment/components/application-assessment-wizard/application-assessment-wizard.tsx

```diff
@@ -0,0 +1,343 @@
+import React, { useEffect, useMemo, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useHistory } from "react-router-dom";
+import { FieldErrors, FormProvider, useForm } from "react-hook-form";
+import { ButtonVariant, Wizard, WizardStep } from "@patternfly/react-core";
+import { AxiosError } from "axios";
+
+import {
+  Assessment,
+  AssessmentStatus,
+  Question,
+  QuestionnaireCategory,
+} from "@app/api/models";
+import {
+  COMMENTS_KEY,
+  getCommentFieldName,
+  getQuestionFieldName,
+  QUESTIONS_KEY,
+} from "../../form-utils";
+import { AssessmentStakeholdersForm } from "../assessment-stakeholders-form";
+import { CustomWizardFooter } from "../custom-wizard-footer";
+import { getApplicationById, patchAssessment } from "@app/api/rest";
+import { formatPath, Paths } from "@app/Paths";
+import { NotificationsContext } from "@app/shared/notifications-context";
+import { getAxiosErrorMessage } from "@app/utils/utils";
+import { WizardStepNavDescription } from "../wizard-step-nav-description";
+import { QuestionnaireForm } from "../questionnaire-form";
+import { ConfirmDialog } from "@app/shared/components";
+
+export const SAVE_ACTION_KEY = "saveAction";
+
+export enum SAVE_ACTION_VALUE {
+  SAVE,
+  SAVE_AND_REVIEW,
+  SAVE_AS_DRAFT,
+}
+
+export interface ApplicationAssessmentWizardValues {
+  stakeholders: number[];
+  stakeholderGroups: number[];
+  [COMMENTS_KEY]: {
+    [key: string]: string; // <categoryId, commentValue>
+  };
+  [QUESTIONS_KEY]: {
+    [key: string]: number | undefined; // <questionId, optionId>
+  };
+  [SAVE_ACTION_KEY]: SAVE_ACTION_VALUE;
+}
+
+export interface ApplicationAssessmentWizardProps {
+  assessment?: Assessment;
+  isOpen: boolean;
+}
+
+export const ApplicationAssessmentWizard: React.FC<
+  ApplicationAssessmentWizardProps
+> = ({ assessment: initialAssessment, isOpen }) => {
+  const { t } = useTranslation();
+
+  const [currentStep, setCurrentStep] = useState(0);
+  const [saveError, setSaveError] = useState<AxiosError>();
```

Comment:
`saveError` is not read anywhere.

Reply:
+1 Looks like this was never used. Will remove.